packages/expo-local-authentication/ios/LocalAuthenticationModule.swift

```diff
@@ -0,0 +1,169 @@
+import ExpoModulesCore
+import LocalAuthentication
+
+public class LocalAuthenticationModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoLocalAuthentication")
+
+    AsyncFunction("hasHardwareAsync") { () -> Bool in
+      let context = LAContext()
+      var error: NSError?
+      let isSupported: Bool = context.canEvaluatePolicy(LAPolicy.deviceOwnerAuthenticationWithBiometrics, error: &error)
+      let isAvailable: Bool = isSupported || error?.code != LAError.biometryNotAvailable.rawValue
+
+      return isAvailable
+    }
+
+      AsyncFunction("isEnrolledAsync") { () -> Bool in
```

Comment:
Oops, looks like still only Name and the first AsyncFunction are indented correctly 😅

Reply:
woops, should be good now :D